OpInfos for torch.{flatten, column_stack}

anjali411 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #69237

Differential Revision: [D32988956](https://our.internmc.facebook.com/intern/diff/D32988956)

anjali411 committed:
OpInfos for torch.{flatten, column_stack}

[ghstack-poisoned]

anjali411 commented:
@anjali411 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32988956).

mruberry commented:
@pytorchbot ciflow rerun -l ciflow/all

mruberry commented on torch/testing/_internal/common_methods_invocations.py:
```diff
@@ -12000,6 +12022,30 @@ def ref_pairwise_distance(input1, input2):
+               # File "pytorch/torch/testing/_internal/common_jit.py", line 306, in test_type
+               # self.assertTrue(actual_type.isSubtypeOf(out_type))
+               # AssertionError: False is not true
+               DecorateInfo(unittest.skip("Skipped!"), 'TestJit', 'test_variant_consistency_jit'),
```
Can this be xfailed?

mruberry commented on torch/testing/_internal/common_methods_invocations.py:
```diff
@@ -12000,6 +12022,30 @@ def ref_pairwise_distance(input1, input2):
+               # please report a bug to PyTorch.
+               DecorateInfo(unittest.skip("Skipped!"), 'TestJit', 'test_variant_consistency_jit', dtypes=(torch.float32,)),
+               # AssertionError: UserWarning not triggered : Resized a non-empty tensor but did not warn about it.
+               DecorateInfo(unittest.skip("Skipped!"), 'TestCommon', 'test_out', dtypes=(torch.float32,)),),
```
Can this be xfailed?

anjali411 replied:
Again seems like something we should fix and is not expected

anjali411 replied:
Again I think this should be fixed and is not expected

mruberry reviewed (approved):
Cool!

Updated with ciflow/all and made two comments to see if we can convert skips to xfails.

anjali411 committed:
Update on "OpInfos for torch.{flatten, column_stack}"


Differential Revision: [D32988956](https://our.internmc.facebook.com/intern/diff/D32988956)

[ghstack-poisoned]

anjali411 committed:
Update on "OpInfos for torch.{flatten, column_stack}"


Differential Revision: [D32988956](https://our.internmc.facebook.com/intern/diff/D32988956)

[ghstack-poisoned]

anjali411 commented:
@anjali411 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32988956).

anjali411 committed:
Update on "OpInfos for torch.{flatten, column_stack}"


Differential Revision: [D32988956](https://our.internmc.facebook.com/intern/diff/D32988956)

[ghstack-poisoned]

anjali411 committed:
Update on "OpInfos for torch.{flatten, column_stack}"


Differential Revision: [D32988956](https://our.internmc.facebook.com/intern/diff/D32988956)

[ghstack-poisoned]

anjali411 committed:
Update on "OpInfos for torch.{flatten, column_stack}"


Differential Revision: [D32988956](https://our.internmc.facebook.com/intern/diff/D32988956)

[ghstack-poisoned]

anjali411 commented:
@anjali411 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32988956).

eellison commented on torch/testing/_internal/common_methods_invocations.py:
```diff
@@ -13183,6 +13205,32 @@ def ref_pairwise_distance(input1, input2):
+               # File "pytorch/torch/testing/_internal/common_jit.py", line 306, in test_type
+               # self.assertTrue(actual_type.isSubtypeOf(out_type))
+               # AssertionError: False is not true
+               DecorateInfo(unittest.skip("Skipped!"), 'TestJit', 'test_variant_consistency_jit'),
```
I'll update the error here but do you mind cc'ing on skipped jit failures / test failures in the future ? this is a legitimate error. admittedly, there are a number of flakey failures, which https://github.com/pytorch/pytorch/issues/69320 should help with.

anjali411 replied:
sorry it's a bit hard to figure out if these jit failures are flaky or real concerns since we already have so many skips. but point taken, will cc in future to cross-check.

eellison replied:
yea, fair. i'll try to prioritize #69320 and other fixes so it's less frequent..
